tests/integration_tests/test_wave_01_02_03.py

```diff
@@ -2,10 +2,10 @@
 from swap_meet.vendor import Vendor
 from swap_meet.item import Item
 
-@pytest.mark.skip
-@pytest.mark.integration_test
+# @pytest.mark.skip
+# @pytest.mark.integration_test
```

Comment:
Please do NOT remove this! Pytest needs to know that this is an integration test. Please only remove the skip above.